src/main/java/mail/MailSender.java

```diff
@@ -0,0 +1,46 @@
+package mail;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.util.Properties;
+
+public class MailSender {
+    private final GeneratorPassword randomPassword = new GeneratorPassword();
+
+    public void sendToEmail(String email, String password) {
+        final String from = "lovejudo96@mail.ru";
+        String host = "smtp.mail.host";
+
+        Properties properties = new Properties();
```

Comment:
Not the best place for configuration properties. 
You will create on every method call